### ISSUE TITLE
LJPME vs dispersion correction

### ISSUE THREAD

**Issue by EqualAPriori** [pandegroup/openmm#2224]
Hi there, thank you all for the wonderful work on openMM!

I'm interested in using PME for computing LJ interactions, and am looking to take advantage of the nonbondedMethod=LJPME option when creating a system.

However, I notice that when I call getUseDispersionCorrection() on the resulting nonbonded force I get "True" back, which seems counterintuitive -- shouldn't the LJPME negate the need for adding the dispersion correction when working with NPT? Does openMM realize that it should ignore the dispersion correction, or do I need to turn it off manually?

Thanks!

**Comment by peastman** [comment]
Yes, the value of that setting is ignored when using LJPME (or when simulating a non-periodic system, of course).
